style_guides/pl_sql_logging.md

```diff
@@ -0,0 +1,35 @@
+## Logging
+
+** To be Updated **
+
+See [Oracle-Base: Instrumenting Your PL/SQL Code](https://oracle-base.com/articles/misc/instrumenting-your-plsql-code) For reference.
+
+Use the `DSP` package to add logging to your code. __DO NOT USE__ use `DBMS_OUTPUT`.

```

Comment:
Just as a reminder for me to discuss, we want to be using DBMS_OUTPUT if we want logs to be output and stored in database jobs log column, in the scheduler runs history table.